packages/theme/scripts/saltIconNames.js

```diff
@@ -17,7 +17,7 @@ const saltIconNames = {
   name: 'iconEnum',
   setup(build) {
     const icons = [];
-    build.onLoad({ filter: /src[\/\\]components/ }, async args => {
+    build.onLoad({ filter: /components/ }, async args => {
```

Comment:
Due to the new style injection the internal structure of the package has changed